pkg/config/config.go

```diff
@@ -828,6 +828,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.remote_tagger", true)
 	bindEnvAndSetLogsConfigKeys(config, "runtime_security_config.endpoints.")
 
+	// Serverless Agent
+	config.BindEnvAndSetDefault("lambda_logs_enabled", true)
```

Comment:
Because we already are supporting `DD_SERVERLESS_FLUSH_STRATEGY` (but not yet through the `config` package though, see [here](https://github.com/DataDog/datadog-agent/blob/master/cmd/serverless/main.go#L84#)), I think it would makes sense to have this one called `serverless_logs_enabled`.

Let me explain why: the library we are using in the `config` package is automatically trying to find configuration values from both environment variables and in the configuration file based on names declared here. For env vars, it uses the pattern: `DD_ + REPLACE(UPPER(configuration_variable_name), ".", "_")` meaning that once it has been declared here, you can either use:

```yaml
serverless.logs_enabled: true
```
or
```yaml
serverless:
  logs_enabled: true
  flush_strategy: "at_end"
```
or use `DD_SERVERLESS_LOGS_ENABLED`.